dashboard/pages/primary_screening/callbacks.py

```diff
@@ -0,0 +1,106 @@
+from dash import html, no_update, callback, Output, Input, State
+
+from ..error import throwable
+from ...data.bmg_plate import parse_bmg_files_from_iostring
+from ...data.file_preprocessing.echo_files_parser import EchoFilesParser
+import io
+import base64
+import functools
+import uuid
+
+
+@throwable(1)
```

Comment:
why use throwable if you do not throw any `DashboardError` in the callback? you do not have to add it when it is not needed

Reply:
btw the 1 in here does not make sense, you return 2 values from the function so if you were to use it you'd have to use 2

please try to avoid copying code if you are not sure what it does and if it will work (pretty sure that it won't work with this setup)